releasenotes/notes/checkconfig-alias-9172271a07db707b.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Make ``agent checkconfig`` an alias of ``agent configcheck``
```

Comment:
```suggestion
    Make `agent checkconfig` an alias of `agent configcheck`.
```

Reply:
I put double back-ticks because the release note comment in the template says that the release notes are in reStructuredText:
```
Each section note must be formatted as reStructuredText.
```

And it seems that literals need double back-ticks in reStructuredText: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#inline-literals

(I should have kept this comment explaining the expected format)

Reply:
Almost every changelog I write gets this review from the docs team -- @apigirl can you perhaps make a general announcement to docs folks that the agent changelog is rST and requires double-backticks?  It'll just save us all a little bit of time :)